### PR TITLE
Created CoC

### PR DESCRIPTION
**Pull request by Arushi282**

### CODE REVIEW

**Review by thayeylolu — APPROVED** [review]
looks good to me

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -6,33 +6,34 @@ We are committed to making participation in this project a harassment-free exper
...
 
 This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
 
-A working group of community members is committed to promptly addressing any reported issues. The working group is made up of pandas contributors and users. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the working group by e-mail (pandas-coc@googlegroups.com). Messages sent to this e-mail address will not be publicly visible but only to the working group members. The working group currently includes
+A working group of community members is committed to promptly addressing any reported issues. The working group is made up of this project's contributors and users. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the working group by e-mail (credit-prediction@googlegroups.com). Messages sent to this e-mail address will not be publicly visible but only to the working group members. The working group currently includes
```
Comment (liannah) [inline CODE_OF_CONDUCT.md:30]: Does this email address exist ?

Comment (Arushi282) [reply CODE_OF_CONDUCT.md]: Hi, just update the email address, please review it and see if it is good to merge, thanks!

**Comment by karanpreetkaur** [comment]
@Arushi282 a little change to add spaces after bullet points as they are not rendering properly in the document otherwise looks good to me.

Just replace (-) with (- )

**Pull request merged**